autoendpoint/src/settings.rs

```diff
@@ -46,6 +46,12 @@ pub struct Settings {
     pub fcm: FcmSettings,
     pub apns: ApnsSettings,
 
+    // These values correspond to the a2 library ClientConfig struct.
```

Comment:
These settings are specific to APNS only. They should be placed in `ApnsSettings`.

Reply:
Np, I'll move them. The ticket had mentioned creating Autoendpoint settings and then passing those values to a2 during router configuration, so I put it in the autoendpoint settings. Does make more sense for them to be with apns, indeed!